src/interceptors/fetch/index.test.ts

```diff
@@ -0,0 +1,96 @@
+import { DeferredPromise } from '@open-draft/deferred-promise'
```

Comment:
I think we should rename this test file to `abort-controller.test.ts` and move it under `test/modules/fetch/compliance` where we store all integration tests.

Reply:
same goes for the test added inside `src/interceptors/ClientRequest/index.test.ts` I assume ?

Reply:
We can leave that one be, it doesn't concern itself with request handling but focuses on how the `.respondWith()` works in the context of the ClientRequest. I think it's fine.

Reply:
I mean I've added a test abort abortion in this file